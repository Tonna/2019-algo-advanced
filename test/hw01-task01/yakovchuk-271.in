30
1 1 1 1 2 4 4 7 1 1 10 6 1 13 6 3 11 13 2 10 9 4 19 23 15 1 26 26 15
5
30 1
26 9
5 7
3 2
30 3
