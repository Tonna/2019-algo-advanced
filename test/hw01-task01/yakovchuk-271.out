13
37
12
4
21
